vmm/src/vm.rs

```diff
@@ -856,6 +848,23 @@ impl Vm {
         )
     }
 
+    pub fn restore_hypervisor_vm(
```

Comment:
nit: Not sure about the function name it doesn't look like it does anything "restore" specific. Could it be more generally named and then reused on the new VM code path too?